src/components/products/ProductCategory.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react'
```

Comment:
Maybe this is not very important, but remember that in the new versions of react there is no need to import React from 'react' in your components.